script/instances-0.6.0-to-0.7.0.sh

```diff
@@ -7,25 +7,40 @@ fi
 
 set -euo pipefail
 
-instanceDir="$1"
-backupDir="$instanceDir.bak_$(date +%s)"
+jq_version="$(jq --version 2>&1)" # redirecting stderr https://github.com/stedolan/jq/issues/1452
+continue=0
+if [[ $jq_version != *"1.5"* ]]; then
+  echo "WARNING: This script was tested against jq-1.5 and should be run against that as well."
+  read -p "WARNING: Looks like you are using $jq_version. Continue anyway? [y/n] " -n 1 -r
```

Comment:
I do not like this behaviour at all; whoever uses this script has no chance to make an informed choice here, as they know even less about how this script uses jq than you (the developer) does.  If you can't make a proper choice here no one else can.

So, please either test the script with lower JQ versions, or just abort if the version is < 1.5 and make the user update their jq version.

That'd also simplify the condition and remove the convoluted "$continue" logic.

If you choose to keep this prompt then at least rewrite it in a way that works without a "$continue" variable that complicates the control flow.